python/lsst/daf/butler/cli/cmd/_remove_runs.py

```diff
@@ -52,25 +66,44 @@ def _print_remove(will: bool, runs: Sequence[str], datasets: Mapping[str, int]):
         The dataset types & count that will be or were removed.
     """
     print(willRemoveRunsMsg if will else didRemoveRunsMsg)
-    print(", ".join(runs))
-    print(willRemoveDatasetsMsg if will else didRemoveDatasetsMsg)
+    unlinkMsg = willUnlinkMsg if will else didUnlinkMsg
+    for run in runs:
+        if run.parents:
+            print(unlinkMsg.format(run=run.name, parents=", ".join(_quoted(run.parents))))
+        else:
+            print(run.name)
```

Comment:
Does the coverage report indicating that this line never runs mean that we aren't testing the command without parents?

Reply:
yes, I changed the tests for CHAINED collections, and didn't consider this case. Will fix.